src/extension.ts

```diff
@@ -106,10 +121,21 @@ export function activate(vsContext: vscode.ExtensionContext) {
 
     vsContext.subscriptions.push(runDisposable);
     vsContext.subscriptions.push(copyDisposable);
+
+    let copySQLDisposable = vscode.commands.registerCommand('extension.copySQL', (sql: string) => {
+        vscode.env.clipboard.writeText(sql).then(() => {
+            vscode.window.showInformationMessage('SQL copied to clipboard!');
```

Comment:
I initially loved to use these notifications, but I suddenly fell into the trap and notifications were spawning everywhere, so I relied on the VS Code guidelines for notifications and is really neat and most of the cases ends in the relax 😎 https://code.visualstudio.com/assets/api/ux-guidelines/examples/notification-decision-tree.png

Reply:
Good point, just removed it.